specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -5483,7 +5483,7 @@
           "items": {
             "type": "string"
           },
-          "description": "Availability Zone to use for this host group � only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
+          "description": "Availability Zone to use for this host group only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
```

Comment:
Can we add a period after "Availability Zone to use for this host group" and make "Only single zone is supported" as a separate statement.

Reply:
Done